src/Components/WelcomeTextForm/WelcomeTextForm.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Formik, Form } from "formik";
+import * as Yup from "yup";
+import { Button, Form as BootstrapForm, FloatingLabel } from "react-bootstrap";
+
+import { postOrganizationData } from "../../Services/organizationService";
+
+const validationSchema = Yup.object({
+  welcomeText: Yup.string()
+    .min(20, "Mínimo 20 caracteres")
+    .required("Campo obligatorio"),
+});
+
+const WelcomeTextForm = ({ welcomeText }) => (
+  <Formik
+    initialValues={{ welcomeText }}
+    validationSchema={validationSchema}
+    onSubmit={(values) =>
+      postOrganizationData({ welcome_text: values.welcomeText, name: "string" })
```

Comment:
No usaria un verbo en el nombre de una funcion de negocio. En un componente deberia poder llamarse a un servicio como bien haces, pero el nombre tiene que ser descriptivo a lo que se espera a realizar a nivel negocio, que puede ser.. updateOrganizationData por ejemplo, o saveOrganizationData. El post es algo especifico de IT

Reply:
Solucionado